downstream/modules/platform/ref-controller-scm-inventory-details.adoc

```diff
@@ -0,0 +1,29 @@
+[id="controller-scm-inventory-details"]
+
+= SCM inventory details
+
+Access the *Details* tab to view details about the job execution and its associated project:
```

Comment:
To view details about the job execution and its associated project, select the Access tab.